src/day1/sorting.c

```diff
@@ -0,0 +1,33 @@
+#include<stdio.h>
```

Comment:
which algorithm is this? what is the complexity? always put comments in your program to tell a bit about your program to the reviewer